week-2/extended-project/valeria_and_friends/jsons/vehicles.json

```diff
@@ -0,0 +1,218 @@
+            "crew": "5", 
+            "passengers": "40", 
+            "cargo_capacity": "1000", 
+            "consumables": "unknown", 
+            "vehicle_class": "assault walker", 
+            "pilots": [], 
+            "films": [
+                "https://swapi.co/api/films/2/", 
+                "https://swapi.co/api/films/3/"
+            ], 
+            "created": "2014-12-15T12:38:25.937000Z", 
+            "edited": "2014-12-22T18:21:15.714673Z", 
+            "url": "https://swapi.co/api/vehicles/18/"
+        }, 
+        {
+            "name": "AT-ST", 
+            "model": "All Terrain Scout Transport", 
+            "manufacturer": "Kuat Drive Yards, Imperial Department of Military Research", 
+            "cost_in_credits": "unknown", 
+            "length": "2", 
+            "max_atmosphering_speed": "90", 
+            "crew": "2", 
+            "passengers": "0", 
+            "cargo_capacity": "200", 
+            "consumables": "none", 
+            "vehicle_class": "walker", 
+            "pilots": [
+                "https://swapi.co/api/people/13/"
+            ], 
+            "films": [
+                "https://swapi.co/api/films/2/", 
+                "https://swapi.co/api/films/3/"
+            ], 
+            "created": "2014-12-15T12:46:42.384000Z", 
+            "edited": "2014-12-22T18:21:15.761584Z", 
+            "url": "https://swapi.co/api/vehicles/19/"
+        }, 
+        {
+            "name": "Storm IV Twin-Pod cloud car", 
+            "model": "Storm IV Twin-Pod", 
+            "manufacturer": "Bespin Motors", 
+            "cost_in_credits": "75000", 
+            "length": "7", 
+            "max_atmosphering_speed": "1500", 
+            "crew": "2", 
+            "passengers": "0", 
+            "cargo_capacity": "10", 
+            "consumables": "1 day", 
+            "vehicle_class": "repulsorcraft", 
+            "pilots": [], 
+            "films": [
+                "https://swapi.co/api/films/2/"
+            ], 
+            "created": "2014-12-15T12:58:50.530000Z", 
+            "edited": "2014-12-22T18:21:15.783232Z", 
+            "url": "https://swapi.co/api/vehicles/20/"
+        }, 
+        {
+            "name": "Sail barge", 
+            "model": "Modified Luxury Sail Barge", 
+            "manufacturer": "Ubrikkian Industries Custom Vehicle Division", 
+            "cost_in_credits": "285000", 
+            "length": "30", 
+            "max_atmosphering_speed": "100", 
+            "crew": "26", 
+            "passengers": "500", 
+            "cargo_capacity": "2000000", 
+            "consumables": "Live food tanks", 
+            "vehicle_class": "sail barge", 
+            "pilots": [], 
+            "films": [
+                "https://swapi.co/api/films/3/"
+            ], 
+            "created": "2014-12-18T10:44:14.217000Z", 
+            "edited": "2014-12-22T18:21:15.807906Z", 
+            "url": "https://swapi.co/api/vehicles/24/"
+        }
+    ]
+}
```

Comment:
The same of https://github.com/ableco/codeable-exercises/pull/26/files#r264268573